src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue

```diff
@@ -111,28 +113,50 @@ const state = reactive({
 });
 
 const deleteModalState = reactive({
-    headerTitle: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.DELETE_TITLE'),
+    type: 'DELETE' as 'DELETE' | 'ESCAPE' | 'CANCEL',
+    headerTitle: '' as string | TranslateResult,
+    contents: '' as string | TranslateResult,
```

Comment:
suggestion: TranslateResult type includes string type :)

```suggestion
    headerTitle: '' as TranslateResult,
    contents: '' as TranslateResult,
```

Reply:
suggestion: If headerTitle, contents are different by type, what about using computed?